src/plugins/blocksmith/utils/getLines.js

```diff
@@ -12,11 +12,13 @@ const getLinesBrokenByLineBreaks = (txt) => {
   let text = txt;
   let matchIndex;
   const lines = [];
+
   while (/\r|\n/.test(text)) {
```

Comment:
@efjacobson - should we add `\r\n` to this group?

Reply:
@arunkarnati not sure what you mean